### PR TITLE
bug 1496246 - Update Pocket CTA implementation with feedback from Leg…

### PR DESCRIPTION
**Pull request by ScottDowne**
…al and UX

### CODE REVIEW

File: locales/en-US/strings.properties [hunk 1]
```diff
@@ -145,6 +145,7 @@ pocket_read_more=Popular Topics:
 pocket_read_even_more=View More Stories
 pocket_more_reccommendations=More Recommendations
 pocket_learn_more=Learn More
```
Comment (k88hudson) [inline locales/en-US/strings.properties]: Is this being used anywhere else? If not, please remove it

Comment (ScottDowne) [reply locales/en-US/strings.properties]: Removed

**Comment by k88hudson** [comment]
Personally I find this colour combination a little weird, is that what's intended?
image

File: content-src/components/PocketLoggedInCta/_PocketLoggedInCta.scss [hunk 2]
```diff
@@ -5,8 +5,8 @@
   margin-inline-end: 20px;
   display: inline-block;
 
-  a {
-    background: $pocket-red;
+  .pocket-cta-button {
```
Comment (k88hudson) [inline content-src/components/PocketLoggedInCta/_PocketLoggedInCta.scss]: Looking at the invision spec, it looks like padding of `0 9px` is closer

**Comment by ScottDowne** [comment]
Updating. I double checked with Aaron, and he said the desired padding was 8px so went with that. Super close to 9 though :)

**Pull request merged**